ui/lib/pki/addon/templates/issuers/generate-root.hbs

```diff
@@ -1 +1,17 @@
-route: issuers.generate-root
\ No newline at end of file
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-pki-key-page-title>
+      Generate root
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<PkiGenerateRoot
+  @model={{this.model}}
+  @onCancel={{transition-to "vault.cluster.secrets.backend.pki.issuers.index"}}
+  @onSave={{transition-to "vault.cluster.secrets.backend.pki.issuers.index"}}
```

Comment:
Would saving take us to the generated root cert?

Reply:
I used to have a note about parsing the issuer from the save response and passing that for the onSave transition. I plan on looking into that with the generate-csr work -- for now, I figured this is a reasonable if not ideal transition

Reply:
Yeesh - the pki workflow flow 🤯. Yeah, sounds more than reasonable!